h2o-py/tests/testdir_misc/pyunit_export_sep.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from os import path
+import pandas as pd
+'''
+Export file with h2o.export_file with a custom separator
+'''
+
+
+def export_custom_separator():
+    data = {'col1': [1, 2], 'col2': [3, 4]}
```

Comment:
I'd suggest trying values with escaped quotes.

Reply:
Can you explain bit more how is it related to this PR?